engine/src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -157,7 +157,11 @@ trait SharedFileSystem { self: Backend =>
   /**
    * Creates host execution directory.
    */
-  def initializeForWorkflow(descriptor: WorkflowDescriptor): Try[Unit] = Try {
+  def initializeForWorkflow(descriptor: WorkflowDescriptor): Future[Unit] = {
+    Future.fromTry(makeExecutionDirectory(descriptor))
```

Comment:
I could but intentionally didn't here - my reasoning was that I wanted to disrupt as few apple carts as possible here. I think that the non-hotfix should make that async though.